### PR TITLE
3.2 eslint implementation

### LINKED ISSUE #17: Linting JS
## Main Goal
Add eslinting

### When Accomplished
- [ ] The linter runs on commands `yarn start` and `yarn build`

## Notes
- [ESLint & CLI](https://eslint.org/docs/user-guide/command-line-interface)

### PR DESCRIPTION
**Pull request by reuelcj**
## What It Does

Add JavaScript linting to the project to ensure formatting and style standards are being met.
close #17

## How To Test

- [x] The linter runs on commands `yarn start` and `yarn build`
- [x] Linting completes with no errors

## Checklist

Under penalty of public shaming, I avow that I:

- [x] Reviewed the diff to look for typos, whitespace errors, and console.log() statements
- [x] Verified that there are no compiler or linter errors or warnings
- [x] Verified the build in production(optimized) mode
- [x] Updated the docs, if necessary
- [x] Included the appropriate labels
- [x] Referenced applicable issues (i.e. Closes #XXXX, Fixes #XXXX)

Browsers tested:
- [x] Chrome
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "extends": "eslint:recommended",
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
```
Comment (kddrake) [inline .eslintrc.js]: you'll want to add rules similar to [these](https://github.com/Banno/platform-ux/blob/master/projects/reports/.lintrc.json#L19-L27). I doubt the rules are a 1-to-1 correlation, but eslint should be able to enforce most, if not all, of them.

File: package.json [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
-  "scripts": { 
-    "start": "http-server . -p 1820 "
+  "scripts": {
+    "start": "http-server . -p 1820",
+    "build": "eslint '**/*.js'" 
+    
   },
-  "dependencies": {
-    "lite-server": "^2.5.3"
+  "devDependencies": {
```
Comment (kddrake) [inline package.json]: make sure you keep `http-server` as apart of your devDependencies, too

File: .eslintrc.js (hunk 1 above)
Comment (reuelcj) [reply .eslintrc.js]: Added.

File: package.json (hunk 2 above)
Comment (reuelcj) [reply package.json]: Done.

File: .eslintrc.js (hunk 1 above)
Comment (reuelcj) [reply .eslintrc.js]: also about the yarn.lock and package-lock.json - with the webpack configuration there are both...

**Pull request merged**